ostelco-ios-client/ViewControllers/Country/AllowLocationAccessViewController.swift

```diff
@@ -91,7 +91,8 @@ class AllowLocationAccessViewController: UIViewController {
                 showLocationAccessRestricted()
             case .denied:
                 showLocationAccessDenied()
-            case .authorizedAlways, .authorizedWhenInUse:
+            case .authorizedAlways,
+                 .authorizedWhenInUse:
```

Comment:
for what it's worth, I find it clearer to put these on separate lines for a couple reasons: 

a) Makes it way more obvious that there is more than one case being handled
b) Makes diffs clearer when additional cases are added or removed to this same handling block.